apps/src/templates/SchoolTypeDropdown.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { Component, PropTypes } from 'react';
+import i18n from "@cdo/locale";
+
+
+const singleLineLayoutStyles = {
+  display: "table-cell",
+  width: 210,
+  verticalAlign: "middle",
+  minHeight: 42,
+  fontSize: 13,
+  fontFamily: '"Gotham 4r", sans-serif',
+  color: "#333",
+  padding: 0,
+};
+const singleLineContainerStyles = {
+  display: "table",
+  width: "100%",
+};
+const selectStyle = {
+  width: 390,
+  verticalAlign: 'top',
+  marginBottom: '5px',
+  marginTop: '5px',
+};
```

Comment:
More often than not I see styles in one styles object, like [this](https://github.com/code-dot-org/code-dot-org/blob/9873b1c7c25f9037123331ac59f242f1afcca6b4/apps/src/templates/studioHomepages/ResourceCard.jsx#L8).

Reply:
+1. This is specified in our [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react)

Reply:
I've seen both. The single object approach seems to be used when there are many styles and the separate ones when there are just a few. I don't have a strong feeling so can change this if you want.

Reply:
Our styleguide (linked above) explicitly specifies the single object, so I'd prefer that. We _shouldn't_ have the other style, and certainly shouldn't be adding more like that.

> Prefer single object for all styles vs. inlined style objects. Define static styles outside the component, and only dynamic styles in the render method.

Reply:
Got it. I'll make the change.